### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adarshxs**
@Narsil

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,10 +90,14 @@ docker run --gpus all --shm-size 1g -p 8080:80 -v $volume:/data ghcr.io/huggingf
...
 
-To see all options to serve your models (in the [code](https://github.com/huggingface/text-generation-inference/blob/main/launcher/src/main.rs) or in the cli:
+To see all options to serve your models (in the [code](https://github.com/huggingface/text-generation-inference/blob/main/launcher/src/main.rs) or in the cli):
 ```
 text-generation-launcher --help
 ```
+For a basic run, you can start the server as follows:
+```shell
+text-generation-launcher --model-id tiiuae/falcon-7b-instruct --num-shard 1 # You can change the model-id accordingly
+```
```
Comment (Narsil) [inline README.md:100]:
```suggestion
```

We're trying not to push too much the local CLI. it has many more caveats in dependencies and hardware.

**Review by Narsil — COMMENTED** [review]
Thanks for the typo fix.

I think we should remove the local CLI example, we're really trying to push the docker CLI instead (which avoids a lot of headaches for dependencies for users).

**Review by Narsil — COMMENTED** [review]
Thanks for the typo fix.

I think we should remove the local CLI example, we're really trying to push the docker CLI instead (which avoids a lot of headaches for dependencies for users).

**Comment by adarshxs** [comment]
That's cool

**Review by Narsil — APPROVED** [review]
Thanks

**Pull request merged**